VoIPController.cpp

```diff
@@ -3031,7 +3036,7 @@ void VoIPController::AddTCPRelays(){
 			relays.push_back(tcpRelay);
 		}
 		for(Endpoint& e:relays){
-			endpoints[e.id]=move(e);
+			endpoints[e.id]=e;
```

Comment:
Calling move is misleading: e is passed as const l-value reference, no move will actually happen